### ISSUE TITLE
Create code that will upload data to data.world

### ISSUE THREAD

**Issue by synflyn28** [Data4Democracy/democratizing_weather_data#3]
Write code that will zip up all generated CSVs and upload the data to data.world.

**Comment by veenaghorakavi** [comment]
If you want any help, I can help.  @synflyn28 What should I do to help?

**Comment by synflyn28** [comment]
Sure @vghorakavi that would be awesome! I'm sort of new to managing projects on Github so what I would do is fork the project and create a feature branch for this issue. Then we can go from there. Thank you so much!

**Comment by veenaghorakavi** [comment]
Done (I created a feature branch on the forked version called feature_branch).  Is there a Slack channel?

**Comment by synflyn28** [comment]
Yes there is! It's under Data4Democracy under #p-weather-data. Nic Colley is the head of the project and I started the repo here. This code is based off of Nic's Code [here](https://trial.dominodatalab.com/u/niccolley/weather/browse?). The general project description was also started by him [here](https://hackpad.com/Democratizing-Weather-Data-D4MPEypvpUl). I would notify him that you are working on this also and he's on Slack as @niccolley. I'm under @jlreid28 on the Data4Democracy Slack as well. Thank you again for the help and I'll be sure to review your pull request once you create it!

**Comment by avhirupc** [comment]
Is this issue resolved?
